config_test.go

```diff
@@ -386,3 +386,62 @@ func TestCustomTag(t *testing.T) {
 	require.Equal(t, "name", s.Name)
 	require.Equal(t, 10, s.Age)
 }
+
+func TestSliceField(t *testing.T) {
+	t.Run("Slice of string - empty", func(t *testing.T) {
+		s := struct {
+			Letters []string `config:"letters"`
+		}{}
+
+		st := store{
+			"letters": "a,b,c",
+		}
+		e := []string{
+			"a",
+			"b",
+			"c",
+		}
+		err := confita.NewLoader(st).Load(context.Background(), &s)
+		require.NoError(t, err)
+		require.EqualValues(t, e, s.Letters)
+	})
+
+	t.Run("Slice of string - non-empty - no appending", func(t *testing.T) {
+
+		s := struct {
+			Letters []string `config:"letters"`
+		}{
+			Letters: []string{"a", "b"},
+		}
+
+		st := store{
+			"letters": "c,d,e",
+		}
+		e := []string{
+			"c",
+			"d",
+			"e",
+		}
+		err := confita.NewLoader(st).Load(context.Background(), &s)
+		require.NoError(t, err)
+		require.EqualValues(t, e, s.Letters)
+	})
+
+	t.Run("Slice of int", func(t *testing.T) {
+		s := struct {
+			Numbers []int `config:"numbers"`
```

Comment:
Does it work on slices of pointers ?

Reply:
Yep it works. I've added a test. Can you tell me if it is relevant ?